packages/@react-types/form/src/index.d.ts

```diff
@@ -14,11 +14,21 @@ import {DOMProps, SpectrumLabelableProps, StyleProps, ValidationState} from '@re
 import {ReactElement} from 'react';
 
 export interface SpectrumFormProps extends DOMProps, StyleProps, SpectrumLabelableProps {
+  /** The contents of the form. */
   children: ReactElement<SpectrumLabelableProps> | ReactElement<SpectrumLabelableProps>[],
+  /** Whether the form elements are displayed with their quiet style. */
   isQuiet?: boolean,
+  /** Whether the form elements are rendered with their emphasized style (blue). */
   isEmphasized?: boolean,
+  /** Whether the form elements are disabled. */
   isDisabled?: boolean,
+  /** Whether user input is required on each of the form elements before form submission. */
   isRequired?: boolean,
+  /** Whether the form elements can be selected but not changed by the user. */
```

Comment:
comma needed? => `selected, but`

Reply:
I noticed this is a sentence used in several places so it was probably decided a while back not to have the comma.